src/test/scala/com/modelfabric/test/HttpEndpointSuiteTestRunner.scala

```diff
@@ -19,6 +19,7 @@ object HttpEndpointSuiteTestRunner {
   val sparqlServerEndpointUserKey = "SPARQL_ENDPOINT_USER"
   val sparqlServerEndpointPasswordKey = "SPARQL_ENDPOINT_PASSWORD"
 
+  // JC: all these authentication, database name should be encapsulated in a SparqlEndpoint or SparqlConnection, similar to JDBC Connection,  class (to be created)
```

Comment:
We have this in the HttpEndpoint. However, I did not want to tie the HttpEndpoint with logic that infers the values from e.g. the environment and has defaults. This is what the test is supposed to do.